pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -1141,12 +1151,10 @@ func calculateTaskRunTimeout(timeout time.Duration, pr *v1beta1.PipelineRun, rpt
 		if pElapsedTime > timeout {
 			return &metav1.Duration{Duration: 1 * time.Second}
 		}
-		timeRemaining := (timeout - pElapsedTime)
-		// Return the smaller of timeRemaining and rpt.pipelineTask.timeout
-		if rpt.PipelineTask.Timeout != nil && rpt.PipelineTask.Timeout.Duration < timeRemaining {
```

Comment:
To make sure I understand what's going on here, the timeout to be set for new taskruns and runs will now just be based on the pipelineTask timeout? If so, this logic can probably be simplified.

Reply:
I'm updating the docstring on a new push shortly - let's see if that clarifies things for you?